ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/editor/codeassist/CodeAssistProcessor.java

```diff
@@ -49,4 +50,10 @@
      * @return an error message or <code>null</code> if no error occurred
      */
     String getErrorMessage();
+
+    /**
+     * The characters that trigger completion automatically.
+     */
+    @Nullable
+    char[] getTriggerCharacters();
```

Comment:
Please consider trigger character sequences of more than one char. See https://github.com/Microsoft/language-server-protocol/issues/138 for details. This means the return type should be `String[]`.